### PR TITLE
avoid to install active and passive vm on the same nodes

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

File: templates/s3_redundant/s3_redundant.py [hunk 1]
```diff
@@ -190,6 +193,8 @@ def install(self):
             passive_data = dict(active_data)
             passive_data['master'] = active_tlog
             passive_data['namespaces'] = namespaces
+            passive_data['excludeNodesVM'] = []
```
Comment (muhamadazmy) [inline templates/s3_redundant/s3_redundant.py]: why on 2 lines, we can have this one line instead
```suggestion
            passive_data['excludeNodesVM'] = [active_dmvm.data['nodeId']]
```

File: templates/s3_redundant/s3_redundant.py [hunk 2]
```diff
@@ -181,6 +182,8 @@ def install(self):
             self.data['activeS3'] = active_s3.name
         active_s3.schedule_action('install').wait(die=True)
         self.logger.info('Installed s3 {}'.format(active_s3.name))
+        active_dmvm = self.api.service.get(template_uid=VM_TEMPLATE_UID, name=self.guid)
```
Comment (serboctor) [inline templates/s3_redundant/s3_redundant.py]: the dm_vm's name is the guid of the `s3` not `s3_redundant`

File: templates/s3_redundant/s3_redundant.py (hunk 1 above)
Comment (zaibon) [reply templates/s3_redundant/s3_redundant.py]: was expecting some comment that will add more to the list. But seems not :-D

File: templates/s3_redundant/s3_redundant.py (hunk 2 above)
Comment (zaibon) [reply templates/s3_redundant/s3_redundant.py]: indeed thx.

**Pull request merged**